client/src/components/highlightChild.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+class HighLightChild extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      view: 0
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    if (this.state.view === 0) {
+      this.setState({
+        view: 1
```

Comment:
You should declare 0 and 1 as constants to put names on them. See this as an example: https://github.com/AirHackers/CheckoutCalendarService/blob/master/client/components/CheckoutCalendar.jsx

Reply:
Created meaningful constant variables.